webserver/app/config.py

```
@@ -0,0 +1,26 @@
+from pydantic.v1 import BaseSettings
+import json
+from typing import Optional
+
+class GCPSettings(BaseSettings):
+    project_id: str
+    zone: str
+    instance_name: str
+
+class Settings(BaseSettings):
+    @classmethod
+    def from_json(cls, file_path: str):
```

Ruff (0.11.9)

Missing return type annotation for classmethod `from_json`

(ANN206)